app/controllers/messages_controller.rb

```diff
@@ -24,7 +24,7 @@ def create
     @message = @assistant.messages.new(message_params)
 
     if @message.save
-      GetNextAiMessageJob.perform_later(@message.conversation.id)
+      GetNextAIMessageJob.perform_later(@message.conversation.id, @assistant.id)
```

Comment:
Deeper in this job, I need to know whether the assistant being used supports images or not (3.5 does not but GPT 4 does). I could have hacked it a different way but I decided to pass in the assistant from here to prepare for the day we let multiple assistants participate in the same conversation